.buildkite/release.yml

```diff
@@ -5,10 +5,7 @@ steps:
   - label: "Run the release"
     key: "release"
     commands: .ci/release.sh | tee release.out
```

Comment:
https://buildkite.com/docs/pipelines/managing-log-output#storing-the-original-log